### PR TITLE
devices: Fixers org-id transition AccountOrOrgTx function.

### PR DESCRIPTION
**Pull request by ldjebran**
## Type of change

What is it?

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [ ] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] I run `make pre-commit` to check fmt/vet/lint/test-no-fdo

### CODE REVIEW

File: pkg/db/db.go [hunk 1]
```diff
@@ -56,9 +56,16 @@ func AccountOrOrg(account string, orgID string, table string) *gorm.DB {
 
 // AccountOrOrgTx returns a gorm db with account or orgID filter from a known gorm db transaction
 func AccountOrOrgTx(account string, orgID string, tx *gorm.DB, table string) *gorm.DB {
+	accountName := "account"
```
[the remark this answers is not in the document]
Comment (ldjebran) [reply pkg/db/db.go:59]: as showed in sqlText

File: pkg/db/db.go [hunk 2]
```diff
@@ -56,9 +56,16 @@ func AccountOrOrg(account string, orgID string, table string) *gorm.DB {
...
 	if table != "" {
-		account = fmt.Sprintf("%s.%s", table, account)
-		orgID = fmt.Sprintf("%s.%s", table, orgID)
+		accountName = fmt.Sprintf("%s.%s", table, accountName)
+		orgIDName = fmt.Sprintf("%s.%s", table, orgIDName)
 	}
-	return tx.Where("((account = ? AND (account != '' AND account IS NOT NULL)) OR (org_id = ? AND (org_id != '' AND org_id IS NOT NULL)))", account, orgID)
+	sqlText := fmt.Sprintf(
+		"((%s = ? AND (%s != '' AND %s IS NOT NULL)) OR (%s = ? AND (%s != '' AND %s IS NOT NULL)))",
+		accountName, accountName, accountName,
```
Comment (mgold1234) [inline pkg/db/db.go:67]: is it should appear 3 times?

**Comment by InsightsDroid** [comment]
Test results are available in [Ibutsu](https://url.corp.redhat.com/ibutsu-runs). The test run IDs are:
ebb30a92-4a52-4f4b-8b3d-f9ecef583e13

File: pkg/db/db.go (hunk 2 above)
Comment (ldjebran) [reply pkg/db/db.go:67]: yes

**Review by AaronH88 — APPROVED** [review]
/lgtm

**Pull request merged**